app/instance.py

```diff
@@ -261,5 +261,53 @@ def internal_server_error(error):
                          {'Content-Type': 'application/vnd.api+json'})
 
 
+@app.errorhandler(422)
+def unprocessableEntityError_handler(error):
+    return make_response(json.dumps({
+            'status': 422,
+            'title': 'Unprocessable Entity'
+        }),
+        422,
+        {
+            'Content-Type': 'application/vnd.api+json'
+        })
+
+
+@app.errorhandler(400)
+def badRequestError_handler(error):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function name "badRequestError_handler" doesn't conform to '[a-z_][a-z0-9_]{1,50}$' pattern](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4712256)